src/webots/nodes/WbDistanceSensor.cpp

```diff
@@ -619,6 +623,7 @@ void WbDistanceSensor::addConfigure(QDataStream &stream) {
   stream << (double)mLut->minValue();
   stream << (double)mLut->maxValue();
   stream << (double)mAperture->value();
+  stream << (double)mRedColorSensitivity->value();
```

Comment:
That seems wrong as the red color sensitivity should not be sent to the controller (and it's not parsed by the libController).
```suggestion
```